src/app/code/community/Shopgate/Cloudapi/Helper/Request.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * Copyright Shopgate Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @author    Shopgate Inc, 804 Congress Ave, Austin, Texas 78701 <interfaces@shopgate.com>
+ * @copyright Shopgate Inc
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License, Version 2.0
+ */
+
+class Shopgate_Cloudapi_Helper_Request extends Mage_Core_Helper_Abstract
+{
+    /**
+     * Parameter indicating a shopgate cloud request
+     */
+    const KEY_SGCLOUD_INAPP = 'sgcloud_inapp';
+
+    /**
+     * @return bool
+     */
+    public function isShopgateRequest()
+    {
+        return Mage::app()->getRequest()->getParam(self::KEY_SGCLOUD_INAPP) === 1;
```

Comment:
I believe it will be a 1 in a string, won't it? I think magento makes all params strings.